material-stepper/src/main/java/com/stepstone/stepper/StepperLayout.java

```diff
@@ -758,7 +770,12 @@ private boolean isLastPosition(int position) {
     }
 
     private Step findCurrentStep() {
-        return mStepAdapter.findStep(mCurrentStepPosition);
+        int rtlPosition = (mStepAdapter.getCount() - 1) - mCurrentStepPosition;
```

Comment:
You are right. Ternary operator is great in this case. I'm just used to if-else syntax for better code readability. Anyway, I have shifted this logic to AbstractFragmentStepAdapter which I believe is a better place for handling RTL position as will be illustrated in a later comment.